CHANGELOG.md

```diff
@@ -30,7 +30,11 @@ Unreleased
     NFC Unicode normalization (\#357)
 -   Updated `data/src/normalize.py` to be more efficient. (\#358)
 
-### Under `wikipron/`
+### Under `wikipron/` and Elsewhere
```

Comment:
"elsewhere", sorry I hate unnecessary titlecase.

Reply:
I copied that 'and Elsewhere' from elsewhere in the CHANGELOG (line 155), but I've lowered it. Do you want me to change the other 'and Elsewhere'?

Reply:
Sorry, n/m, if you want to you can fix both in a subsequent change.